README.md

```diff
@@ -16,7 +16,27 @@ the square brackets (for multiple-choice questions).
 What does it mean for an HTML tag to be "semantic"? Give an example of a short snippet of HTML written using semantic tags and non-semantic tags.
 
 ```text
-Your answer here.
+A semantic element is tagged with a tag that means something inherently to both browser and developer. Meaning, a header has
+an inherent function, it serves as a header. Another example would be a <nav> tag. It has inherent properties within HTML. This is opposed to a non-semantic tag which
+is defined by the developer. This would be something defined as "div" or "container."
```

Comment:
Semantic tags specifically tell you the function of the text inside them. *All* standard tags have inherent properties within HTML, and you can define your own tags, too, semantic and non-semantic (we'll do that in Angular).